backend/routes.py

```
@@ -93,3 +98,32 @@ async def github_webhook(request: Request):
+        issues = await issue_service.fetch_beginner_issues(
+            owner=GITHUB_ORG,
+            repo=repo
+        )
+
+        return {
+            "repo": repo,
+            "count": len(issues),
+            "issues": issues
+        }
+
+    except Exception as e:
```

Ruff (0.15.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)